index.js

```diff
@@ -1,7 +1,124 @@
+const b = hacker2;
+
+if (a.localeCompare(b) === -1) {
+  console.log(`The driver's name goes first.`);
+} else if (a.localeCompare(b) === 1) {
+  console.log(`Yo, the navigator goes first, definitely.`);
+} else {
+  console.log(`What?! You both have the same name?`);
+}
+
+//Bonus 1
+
+function getWordCount(str) {
+  return str.split(" ").filter(function (n) {
+    return n != "";
+  }).length;
+}
+
+const longText =
+  "Lorem ipsum dolor sit adipiscing elit. Integer non justo neque. Morbi suscipit et sapien non vulputate. Vivamus eu laoreet nisl. Suspendisse a sodales lorem. Etiam volutpat lacus a tincidunt vulputate. Ut tincidunt quis nisi a rutrum. Integer id sem nisl. Ut sodales congue ornare. Suspendisse a sem rhoncus lorem hendrerit fermentum. Etiam quis porttitor nulla. Vestibulum quis consectetur felis, vel consequat dolor. Nam ut mauris euismod, interdum ipsum bibendum, porttitor ex. Nulla eu pulvinar eros. Proin sit amet lobortis libero. Aliquam volutpat commodo magna, eget feugiat nunc volutpat ac. Nam tempus in lacus non pharetra. Sed consequat ex at sem elementum egestas. Aliquam aliquam, tellus nec elementum pretium, nisi lacus bibendum mauris, ultricies ultrices leo turpis bibendum mi. Aliquam lobortis aliquet consectetur. Nunc facilisis ante neque, eu tempor dolor consectetur dapibus. Fusce et scelerisque velit. Proin ac ipsum at risus consequat iaculis in a justo. Sed posuere tellus et luctus tristique. Vivamus dapibus egestas purus sed interdum. Mauris ac augue erat. Praesent interdum volutpat nulla nec luctus. Vivamus a odio hendrerit, volutpat nisi dapibus, pulvinar lorem. Etiam a mi tincidunt, vestibulum lectus eget, hendrerit augue. Nulla mi libero, bibendum non felis eget, ullamcorper varius dui. Integer sit amet augue nec nibh rhoncus ultrices. Proin erat orci, dapibus eu commodo in, eleifend ac massa. Donec enim ante, dictum eu pulvinar nec, suscipit at sapien. Phasellus nunc enim, dictum nec scelerisque non, accumsan nec sem. Quisque vitae elementum mauris. Suspendisse id nibh porttitor, dapibus erat quis, posuere est. Maecenas pretium dui justo, quis pellentesque nibh feugiat eu. Nulla et orci quis sem ultrices vulputate. Nulla nec posuere mauris, at molestie purus. Pellentesque mauris ipsum, elementum vitae ex id, sodales mollis augue. Pellentesque habitant morbi tristique senectus et netus et malesuada fames ac turpis egestas. Vestibulum ante ipsum primis in faucibus orci luctus et ultrices posuere cubilia curae; Cras faucibus convallis dolor sit amet lacinia.amet, consectetur";
+
+console.log("Total words: " + getWordCount(longText));
+
+let count = 0;
+
+for (let i = 0; i < longText.length; i++) {
+  const twoChars = longText[i] + longText[i + 1];
+
+  if (twoChars === "et") {
+    count++;
+  }
+}
+
+console.log(`The Latin word "et" appears ${count} times.`);
+
+//Bonus 2 (Don´t know how to remove space in between)
+
+let phraseToCheck = "taco cat";
+let phraseToCheckTrimmed = phraseToCheck.trim();
```

Comment:
The .trim() removes the white space from start and end, that's why it didn't solve your problem here.